### DISCUSSION TITLE
Sync PR merge config by @oscard0m

### DISCUSSION THREAD

**Discussion by oscard0m** [octoherd/octoherd#22] [Show and tell]
I just created an Octoherd script to synchronize merge options from a repository used as a template and applying those settings to a list of repositories.

Inspired by:
- @gr2m https://github.com/octoherd/script-sync-branch-protections
- @MunifTanjim https://github.com/MunifTanjim/octoherd-script-set-pr-merge-config
- Google's Probot App: https://github.com/googleapis/repo-automation-bots/tree/master/packages/sync-repo-settings

Before merging the script to `main` branch I would love some feedback from you if you don't mind ❤️ 👍🏽 :
https://github.com/oscard0m/octoherd-script-sync-repo-settings/pull/1

---

Also, when creating this script and checking existing solutions, a Discussion Topic come up to me 💡 . I invite you to join and discuss there: https://github.com/octoherd/octoherd/discussions/21

**Comment by gr2m** [comment]
ping @bcoe ☝🏼 

Could you add the "Automatically delete head branches" setting, or is it not yet supported by the APIs?

**Comment by gr2m** [comment]
Just ran it against all @octokit repositories and it worked great so far :)
